lib/Solver/Z3Solver.cpp

```diff
@@ -87,7 +87,7 @@ Result<SolverErrorCode> Z3Solver::addFeature(const string &FeatureName) {
       OptionToVariableMapping.end()) {
     return ALREADY_PRESENT;
   }
-  z3::expr Feature = Context.bool_const(FeatureName.c_str());
+  z3::expr const Feature = Context.bool_const(FeatureName.c_str());
```

Comment:
```suggestion
  const z3::expr Feature = Context.bool_const(FeatureName.c_str());
```
We use west `const` in the code base

Reply:
Thanks for the info. I will change this in my code base. :+1: